uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_lists.scss

```diff
@@ -3,6 +3,10 @@
   margin: 0;
   padding: 0;
 
+  > div {
```

Comment:
what's the markup here? You don't have a class you can target?

Reply:
Fixed by adding `ct-no-result` class.